src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/UpdateOrganisationValidatorImpl.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.professionalapi.controller.request;
+
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.professionalapi.domain.*;
+import uk.gov.hmcts.reform.professionalapi.persistence.OrganisationRepository;
+
+
+@Service

```

Comment:
Replaced with  @Component so that it gets picked up in Controller's all arg Constructor.

Reply:
No. I don't understand why you would want to inject this class and maintain it via Spring? Your validation is essentially a set of static methods with known inputs and outputs. You should just instantiate a normal class or create a utility method with static methods that gets used in the REST Controller code.

Reply:
1) Controller has AllArgsConstructor with service and 2 validators
2) Spring will  require to initialise the dependencies in allArgConstructors, and if validators are not tagged with any Spring annotation then it would fail to get dependency
3)Its good to use Spring depedency creation which makes class singleton rather than creating new objects manually.